### PR TITLE
feat: add date to form submission export

### PR DESCRIPTION
**Pull request by neatbyte-vnobis**
## Changes
Implements issue "Add date/time to the export"

## How Has This Been Tested?
Manual

## Documentation
None

### CODE REVIEW

File: packages/api-form-builder/src/plugins/graphql/form.ts [hunk 1]
```diff
@@ -549,6 +549,12 @@ const plugin: GraphQLSchemaPlugin<FormBuilderContext> = {
                         for (let i = 0; i < submissions.length; i++) {
                             const submissionData = submissions[i].data;
                             const row: Record<string, string> = {};
+
+                            row["Date submitted"] = new Date(submissions[i].createdOn)
```
Comment (adrians5j) [inline packages/api-form-builder/src/plugins/graphql/form.ts]: Let's use [`date-fns`](https://date-fns.org/) for this.

I saw it [already being used](https://github.com/webiny/webiny-js/blob/dev/packages/db-dynamodb/src/plugins/definitions/DateTimeTransformPlugin.ts) on the backend, so it should not additional size on disk.

**Review by adrians5j — CHANGES_REQUESTED** [review]
One minor thing.

Thanks!

**Comment by SvenAlHamad** [comment]
I've tested this and it's working. However let's just adjust the label so it says "Date submitted (UTC)". Later down the line we'll see how we make sure the dates are translated to the user's timezone.

**Review by adrians5j — APPROVED** [review]
Thanks!

**Pull request merged**